UIMentoringSession/MenteesChallenges/Start Screen UI Challenge/Bilyk/MaksymBilyk_StartScreenViewController.xib

```diff
@@ -0,0 +1,75 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<document type="com.apple.InterfaceBuilder3.CocoaTouch.XIB" version="3.0" toolsVersion="21701" targetRuntime="iOS.CocoaTouch" propertyAccessControl="none" useAutolayout="YES" useTraitCollections="YES" useSafeAreas="YES" colorMatched="YES">
+    <device id="retina6_12" orientation="portrait" appearance="light"/>
+    <dependencies>
```

Comment:
Wrong font-weight and font-size for button
![Uploading Screenshot 2023-05-27 at 09.34.45.png…]()